build.gradle

```diff
@@ -129,7 +129,8 @@ rat {
        "bigtop-deploy/puppet/README.md",
        "bigtop-deploy/puppet/modules/qfs/README.md",
        "bigtop-deploy/puppet/config/site.csv.example",
-       "dl/**"
+       "dl/**",
+       "bigtop-deploy/puppet/modules/gpdb/templates/**"
```

Comment:
please also add this exclude to the pom.xml file - we still use the old maven based build for some release tasks, but slowly getting rid of it.